website/addons/dryad/model.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+
+from modularodm import fields
+
+from website.addons.dryad import utils as dryad_utils
+from website.addons.base import AddonUserSettingsBase, AddonNodeSettingsBase
+
+
+class AddonDryadUserSettings(AddonUserSettingsBase):
+    """
+     Stub in place for Dryad addon v2 upgrade.
+
+     As migrations are a little tricky, this stub is being left in since a user
+     settings class will be needed in the future.
+    """
+    has_auth = True
+
+
+class AddonDryadNodeSettings(AddonNodeSettingsBase):
```

Comment:
The switch to v2 using their (currently unstable) SWORD API will likely be simplified by using `AddonOAuthNodeSettingsBase`, maybe `StorageAddonBase` as well. This doesn't need to happen now, as it will require some extra finagling, but would simplify things down the line. Consider switching, but not a blocker